### PR TITLE
Adding messages for automatic tab translation

### PR DESCRIPTION
**Pull request by andrenatal**

### CODE REVIEW

**Comment by flodolo** [comment]
Can you show me how these strings are used? I'm confused on how/when these two strings are displayed

**Comment by andrenatal** [comment]
Screen Shot 2022-04-07 at 9 06 42 AM
Screen Shot 2022-04-07 at 9 06 52 AM

File: en_US/messages.json [hunk 1]
```diff
@@ -116,5 +116,13 @@
     "languageDefaultOption": {
       "message": "Choose a language",
       "description": "Default option in the language dropdown."
+    },
+    "translateAsBrowseOn": {
+      "message": "Translate this tab automatically as you browse?",
+      "description": "Message displayed to the user if they want to turn on automatic translation in the tab."
+    },
+    "translateAsBrowseOff": {
+      "message": "Turn off automatic translation as you browse",
```
Comment (flodolo) [inline en_US/messages.json]: Should this still mention tab?

File: en_US/messages.json [hunk 2]
```diff
@@ -116,5 +116,13 @@
     "languageDefaultOption": {
       "message": "Choose a language",
       "description": "Default option in the language dropdown."
+    },
+    "translateAsBrowseOn": {
+      "message": "Translate this tab automatically as you browse?",
+      "description": "Message displayed to the user if they want to turn on automatic translation in the tab."
```
Comment (flodolo) [inline en_US/messages.json]:
```suggestion
      "description": "Message displayed to the user (as a button), asking if they want to turn on automatic translation for the tab."
```

File: en_US/messages.json [hunk 3]
```diff
@@ -116,5 +116,13 @@
...
+    },
+    "translateAsBrowseOff": {
+      "message": "Turn off automatic translation as you browse",
+      "description": "Message displayed to the user if they want to turn off automatic translation in the tab."
```
Comment (flodolo) [inline en_US/messages.json]:
```suggestion
      "description": "Message displayed to the user (as a button) to turn off automatic translation in the tab."
```

**Review by flodolo — APPROVED** [review]
Thanks for the screenshots. 

I don't think it's a very common UX to have a button with a question mark?

In case, let me know if this should be merged.

File: en_US/messages.json (hunk 1 above)
Comment (andrenatal) [reply en_US/messages.json]: sure, I implemented both suggestions. thanks!

**Comment by andrenatal** [comment]
yes, we need to merge it.

**Review by flodolo — APPROVED** [review]

**Pull request merged**